scripts/reducers/sankey/getChoropleth.js

```diff
@@ -0,0 +1,56 @@
+import _ from 'lodash';
+
+export default function(selectedVectorLayers, nodesDictWithMeta, allColors) {
+  const horizontalLayer = selectedVectorLayers.horizontal;
+  const verticalLayer = selectedVectorLayers.vertical;
+  const isBidimensional = horizontalLayer.uid && verticalLayer.uid ? true : false;
+  const isEmpty = !horizontalLayer.uid && !verticalLayer.uid;
+  const isHorizontal = !!horizontalLayer.uid;
+
+  const geoNodes = _.filter(nodesDictWithMeta, node => node.geoId !== undefined);
+  const nodeIds = Object.keys(geoNodes);
+  const choropleth = {};
+
+  nodeIds.forEach(nodeId => {
+    const node = geoNodes[nodeId];
+    let color = 'none';
+
+    if (isEmpty) {
+      color = 'none';
+    } else if (!node.meta) {
+      color = '#0f0';
```

Comment:
I think literal colours should be moved to constants, this way we control theme easily and they are located in one place.